### PR TITLE
Added my photo 

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Added my photo and updated yml

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -188,3 +188,13 @@
...
   
+- name: Arjun Pandey
+  country: India
+  github: arjunpandey2003
+  image: arjun.jpg
+  twitter: arjunpandey2003
+  lat: 77.2090
+  lng: 28.6139 
+  
+
```
Comment (abishekvashok) [inline _data/students.yml]: Unwanted blank lines, please remove them

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: _data/students.yml (hunk 1 above)
Comment (arjunpandey2003) [reply _data/students.yml]: done

**Review by sansyrox — APPROVED** [review]

File: _data/students.yml [hunk 2]
```diff
@@ -188,3 +188,10 @@
   image: Krish.jpg
   twitter: krishbirla13
   
+- name: Arjun Pandey
+  country: India
+  github: arjunpandey2003
+  image: arjun.jpg
+  twitter: arjunpandey2003
+  lat: 77.2090
+  lng: 28.6139
```
Comment (sayak119) [inline _data/students.yml:197]: leave a single empty line at the end of the file. Otherwise LGTM

**Review by sayak119 — CHANGES_REQUESTED** [review]

**Review by nikit19 — APPROVED** [review]

File: _data/students.yml (hunk 2 above)
Comment (arjunpandey2003) [reply _data/students.yml:197]: its already there

**Review by anshumanv — APPROVED** [review]
LGTM

**Review by abishekvashok — APPROVED** [review]
ack

**Pull request merged**